packages/components/src/components/money/money.tsx

```diff
@@ -30,7 +30,7 @@ const Money = ({
     return (
         <React.Fragment>
             {has_sign && sign}
-            <span className={className}>
+            <span data-testid='dt_span' className={className}>
```

Comment:
let's pass `data-testid` directly to this component, otherwise if you have multiple usage of it in the same page, then you may have issues with your tests. we had this problem with another component.
you can update `TMoneyProps` like this: 
```
type TMoneyProps = React.HTMLAttributes<HTMLDivElement> &{
    amount: number | string;
    className: string;
    currency: string;
    has_sign: boolean;
    should_format: boolean;
    show_currency: boolean; // if true, append currency symbol
};
```
and then pass `data-testid='your_id'` to your component;

Reply:
@shayan-deriv this is the main component where should I pass that prop? :smile:

Reply:
I guess we don't need that right now

Reply:
you should pass it wherever we are using this component (if needs to have `testid`). but we need to change it :smile: . it's not correct to have it the way it is now.

Reply:
if it's not causing any issues right now, we can change them later with other ones.